articles/quickstarts/index.md

```diff
@@ -19,17 +19,146 @@ Learn how to install the Microsoft Quantum Development Kit (QDK), so that you ca
+
+### [Install using conda (recommended)](#tab/tabid-conda)
+
+1. Install [Miniconda](https://docs.conda.io/en/latest/miniconda.html) or [Anaconda](https://www.anaconda.com/products/individual#Downloads). **Note:** 64-bit installation required.
+
+1. Open an Anaconda Prompt.
+
+   - Or, if you prefer to use PowerShell or pwsh: open a shell, run `conda init powershell`, then close and re-open the shell.
+
+1. Create and activate a new conda environment named `qsharp-env` with the required packages (including Jupyter Notebook and IQ#) by running the following commands:
+
+    ```
+    conda create -n qsharp-env -c quantum-engineering qsharp notebook
+
+    conda activate qsharp-env
+    ```
+
+1. Run `python -c "import qsharp"` from the same terminal to verify your installation and populate your local package cache with all required QDK components.
+
+### [Install using .NET CLI (advanced)](#tab/tabid-dotnetcli)
+
+1. Prerequisites:
+
+    - [Python](https://www.python.org/downloads/) 3.6 or later
+    - [Jupyter Notebook](https://jupyter.readthedocs.io/en/latest/install.html)
+    - [.NET Core SDK 3.1](https://dotnet.microsoft.com/download/dotnet-core/3.1)
+
+1. Install the `Microsoft.Quantum.IQSharp` package.
+
+    ```dotnetcli
+        dotnet tool install -g Microsoft.Quantum.IQSharp
+        dotnet iqsharp install
+    ```
+
+    > [!NOTE]
+    > If you get an error during the `dotnet iqsharp install` step, open a new terminal window and try again.
+    > If this still doesn't work, try locating the installed `dotnet-iqsharp` tool (on Windows, `dotnet-iqsharp.exe`) and running:
+    > ```
+    > /path/to/dotnet-iqsharp install --user --path-to-tool="/path/to/dotnet-iqsharp"
+    > ```
+    > where `/path/to/dotnet-iqsharp` should be replaced by the absolute path to the `dotnet-iqsharp` tool in your file system.
+    > Typically this will be under `.dotnet/tools` in your user profile folder.
+    
+***
+
+You now have the IQ# kernel for Jupyter, which provides the core functionality for compiling and running Q# operations from Q# Jupyter Notebooks.
+
+## Use the QDK Online
+
+You can also develop Q# code without installing anything locally with these options:
+
+|Resource|Advantages|Limitations|
+|---|---|---|
+|**Visual Studio Codespaces**|A rich online development environment  |Requires an Azure subscription and plan |
+|**MyBinder** | Free online notebook experience |No persistence |
+
+
+### Using Visual Studio Codespaces
+
+To use Visual Studio Codespaces for Q# development:
+
+1. Create an [Azure account](https://azure.microsoft.com/free/).
+2. Create a Codespaces environment, following the [quickstart guide](https://docs.microsoft.com/visualstudio/codespaces/quickstarts/browser). When creating your codespace, we recommend using *microsoft/quantum* in the **Git Repository** field to load QDK-specific settings.
```

Comment:
@cgranade, since the template is on a private repo (and a preview), I don't think we can point to it directly here, but we can specify that the user can use microsoft/quantum to get started with the samples or any other of repository.

Reply:
To clarify, I don't mean to point to that repo in particular, but as an example of a repo that has the right metadata to support new projects. Codespaces needs there to be a .devcontainer/devcontainer.json file specifying how the new codespace is configured, such that using Codespaces with a new blank repository won't work for Quantum Development Kit usage. At the other extreme, starting with microsoft/quantum will set users up for using the Quantum Development Kit samples and is not ideal for creating new projects (indeed, we've seen a lot of users get started with the QDK in the past by modifying samples _in situ_, rather than copying sample code into their own projects).

To solve both problems, we could use GitHub's template repository functionality to make it easy to create new repositories preconfigured for QDK usage. The https://github.com/cgranade/quantum-research-template repo is an example of how to do so.